core/src/main/java/curso/input/Persistencia.java

```diff
@@ -8,12 +8,14 @@ public class Persistencia implements InterfacePersistence{
     private ArrayList<Course> cursos = new ArrayList<Course>();
 
     @Override
-    public void saveCourse(Course course) {
+    public boolean saveCourse(Course course) {
         if (existCourse(course.getNombre())) {
             throw new exceptionCursoIncompleto("Ya existe el curso");
         }
         cursos.add(course);
+        return true;
```

Comment:
Qué devolvemos si cursos.add() "falla"? En este caso, ya que el UseCase espera un boolean, devolveríamos ese resultado.
```suggestion
        return cursos.add(course);
```